cmd/crio/main.go

```diff
@@ -58,8 +58,9 @@ func validateRuntimeConfig(config *server.Config) error {
 	if config.RuntimeConfig.Runtime != "" {
 		logrus.Warn("runtime is deprecated in favor of runtimes, please switch to that")
 	}
-	if len(config.RuntimeConfig.Runtimes) == 0 && config.RuntimeConfig.Runtime == "" {
-		return errors.New("no deafult runtime configured")
+	_, ok = config.RuntimeConfig.Runtimes[config.RuntimeConfig.DefaultRuntime]
+	if !ok && config.RuntimeConfig.Runtime == "" {
```

Comment:
This check could be moved above in the code, since I think that's an important check that you want to perform even before you check for any untrusted runtime.
Also, please make sure this is included in the commit introducing `DefaultRuntime`.

Reply:
I'll squash them before merging, no worry :)